lib/node_modules/@stdlib/math/base/special/minmaxf/src/main.c

```diff
@@ -0,0 +1,63 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/minmaxf.h"
+#include "stdlib/math/base/assert/is_nanf.h"
+#include "stdlib/math/base/assert/is_negative_zerof.h"
+
+/**
+* Evaluates the single-precision min and max value.
+*
+* @param x      first input
+* @param y      second input
+* @param out 	output array
+* @return		void
+*
+* @example
+* float out = [ 0.0, 0.0 ];
+* double v = stdlib_base_minmaxn( 3.2, 4.1, out );
+*
+* @example
+* double out = [ 0.0, 0.0 ];
+* double v = stdlib_base_minmaxn( 4.0, 0.0/0.0, out );
+*/
+void stdlib_base_minmaxf( const float x, const float y, float * out ){
```

Comment:
@MeastroZI Instead of this, we should have the following function signature for this package:

```c
void stdlib_base_minmaxf( const float x, const float y, float* min, float* max )
```
Then, we can store the minimum and maximum values in `min` and `max` respectively.
Can you please update this here and accordingly everywhere in this PR?